### PR TITLE
P0 - Alert and button

### PR DESCRIPTION
**Pull request by rishmi5h**
Created
preview : https://deploy-preview-3--sigma-ui.netlify.app/pages/docs.html#alert
- Alerts
image

image

- Buttons

image

image

image

### CODE REVIEW

**Review by sonishreyas — APPROVED** [review]
Looks good to me

File: pages/docs.html [hunk 1]
```diff
@@ -1,287 +1,295 @@
...
+      </button>
+      <button class="nav-section-items btn">
+        <a class="link-no-style" href="docs.html">Docs</a>
+      </button>
+    </div>
+    <div class="nav-section">
+      <i class="fas fa-moon nav-section-items theme-switch"></i>
+      <a class="link-no-style" target="_blank" href="https://www.linkedin.com/in/rishmish/"><i
+          class="fab fa-linkedin nav-section-items"></i></a>
+      <!-- <a
```
Comment (Rockyraven) [inline pages/docs.html]: remove unwanted code

File: pages/docs.html [hunk 2]
```diff
@@ -1,287 +1,295 @@
...
           class="link-no-style"
           target="_blank"
           href="https://twitter.com/rishabhmishrabu"
           ><i class="fab fa-twitter nav-section-items"></i
         ></a> -->
```
Comment (Rockyraven) [inline pages/docs.html:44]: remove unwanted code

File: pages/docs.html [hunk 3]
```diff
@@ -234,8 +234,7 @@ <h3 class="m-b-2">Example of Badge on Icons</h3>
         <div class="component-intro m-t-4 m-b-2">
           <h1 class="m-b-2">Image</h1>
           <p>
-            Badges are being used to display a notification count or status
-            information
+            Images can be responsive and also can be customised to be round shaped
```
Comment (Rockyraven) [inline pages/docs.html]: put that line commetted

File: styles/components/button.css [hunk 4]
```diff
@@ -15,6 +9,63 @@
...
+  box-shadow: 0 2px 8px var(--black);
+}
+
+.primary-outline-btn {
+  border: 1px solid var(--primary-color);
+  padding: 0.5rem 2rem;
+  border-radius: 6px;
+  color: var(--primary-color);
+  background-color: var(--white);
+  box-shadow: 12px 12px 16px 0 #00000040, -8px -8px 12px 0 #ffffff4d;
```
Comment (Rockyraven) [inline styles/components/button.css:33]: put that color in var

File: styles/components/button.css [hunk 5]
```diff
@@ -15,6 +9,63 @@
...
+.icon-btn:hover {
+  color: var(--primary-color);
+}
+
+.floating-action-btn {
+  background-color: var(--black);
+  padding: 0.5rem;
+  border-radius: 50%;
+  color: var(--white);
+  box-shadow: 12px 12px 16px 0 #00000040, -8px -8px 12px 0 #ffffff4d;
```
Comment (Rockyraven) [inline styles/components/button.css:74]: here to

File: styles/navbar.css [hunk 6]
```diff
@@ -17,6 +18,21 @@ nav {
   /* text-align: center; */
```
Comment (Rockyraven) [inline styles/navbar.css:18]: remove unwnated code

**Pull request merged**